packages/react-component-library/package.json

```diff
@@ -114,8 +114,6 @@
     "eslint-config-airbnb": "^18.0.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jest": "^23.6.0",
```

Comment:
I'm not sure why these are all repeated here when they're hard dependencies of `@defencedigital/eslint-config-react`. I've removed these two as it was leading to inconsistent versions being used. (Alas, the same applies to the other ones I haven't touched...)